src/book/02-projects/03-hardware/01-overview.mdx

```diff
@@ -6,9 +6,34 @@ description: Overview and specification of the hardware used by NUbots.
+NUbots uses the NUgus hardware platform, a modification of the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling.
 
-![Coronal image of the NUgus robot](../../../images/front-robot.png)
-![Sagittal image of the NUgus robot](../../../images/side-robot.png)
+![Sagittal image of the NUgus robot](../../../images/nugus-side.png 'Sagittal image of the NUgus robot')
+![Coronal image of the NUgus robot](../../../images/nugus-front.png 'Coronal image of the NUgus robot')
 
-The arms and head of the NUgus robot use [MX64AR servos from Dynamixel](http://www.robotis.us/dynamixel-mx-64ar/). The legs use [MX106 servos from Dynamixel](http://support.robotis.com/en/product/actuator/dynamixel/mx_series/mx-106.htm). The robots use two [FLIR Blackfly S](https://flir.app.boxcn.net/s/x0r0bdsjstdkudjd2dwlxoa5fdvj2gpg/file/418603726810) cameras positioned in their head. In the torso lies the Intel NUC7i7bnh, 8GB RAM, 256GB SSD, Robotis CM740 subcontroller, and a 3850mAh 4s LiPo battery.
+## igus® Modifications
+
+The following modifications were made to fit our specific requirements:
+
+- The components of the robot were split to to fit the size of the printer bed and volume.
+- The torso was modified to have open shoulder covers for shoulder servo cable access.
+
+## Servos
+
+The arms and head of the NUgus robot use [MX64AR servos from Dynamixel](http://www.robotis.us/dynamixel-mx-64ar/), with two in each shoulder, one in each elbow, and two in the neck. The legs use [MX106 servos from Dynamixel](http://support.robotis.com/en/product/actuator/dynamixel/mx_series/mx-106.htm), with two in each ankle, three in each hip, and one in each knee.
+
+## Cameras
+
+The NUgus robot uses two [FLIR Blackfly S](https://flir.app.boxcn.net/s/x0r0bdsjstdkudjd2dwlxoa5fdvj2gpg/file/418603726810) USB3 cameras with a C-Mount lens positioned in their head, producing images at 170FPS. Two custom-built S-Mount to C-Mount lens adaptors are used to attach two [Lensation 180° fisheye S-Mount lenses](https://www.lensation.de/product/BF10M19828S118C/). Two [20" USB3 micro USB to up angle USB A cables](https://www.angledcables.com/up-angle-a2.html) are used to connect the cameras to the robot's computer.
+
+## Computing
+
+The NUgus robot uses the [Intel NUC7i7bnh](https://ark.intel.com/content/www/us/en/ark/products/95065/intel-nuc-kit-nuc7i7bnh.html) NUC with a [Silverstone NT07-115X](https://www.silverstonetek.com/product.php?pid=442&area=en) cooling fan. It has 8GB of RAM, a 256GB SSD, and a [Robotis CM740 Sub Controller](<http://support.robotis.com/en/product/robotis-op2/sub_controller(cm-740).htm>).
```

Comment:
Is it 8GB or RAM?

Reply:
https://www.pccasegear.com/products/33137/corsair-cmso8gx4m2a2133c15-8gb-2x4gb-ddr4-sodimm

Reply:
The cooling fan also required custom modifications. The supporting arms were cut off and extra screw holes were drilled to line up with the mounting points on the NUC. A copper spacing plate was also milled to bridge the gap between the NUC processor and the fan.

Reply:
Custom fan to power cable, custom pcb for CM740 -> NUC power, Custom USB NUC to -> CM740 cable

Reply:
USB and fan sense connector are Molex 53398-0471 (1.25 mm pitch PicoBlade header, surface mount, vertical, lead-free, 4 circuits).
Power connectors (on the NUC and fan) are Molex 50-37-5043 (2.50mm Pitch, Mini-SPOX Receptacle Crimp Housing, Single Row, Friction Lock, 4 Circuits, Natural) (this is the [standard 4-pin dynamixel connector](http://support.robotis.com/en/product/actuator/dynamixel/dxl_connector.htm)

CM740->NUC power is a [Yeonho SMH250-02](http://www.yeonho.com/cn/product/product_view.asp?part=device&page=&idx=128&listview=list2) (this is a impossibly hard to get part)

Reply:
These ones for the connectors? https://au.element14.com/molex/53398-0471/connector-header-4pos-1row-1-25mm/dp/2426363
https://au.element14.com/molex/50-37-5043/connector-rcpt-4pos-1row-2-5mm/dp/9979565

Reply:
The second one looks correct, I don't think the first one is

Reply:
Should be this one
https://au.element14.com/molex/51021-0400/connector-housing-rcpt-4pos/dp/9733116?MER=sy-me-pd-mi-acce